meta-lmp-base/conf/layer.conf

```diff
@@ -30,6 +30,8 @@ BBFILES_DYNAMIC += " \
     parsec-layer:${LAYERDIR}/dynamic-layers/parsec-layer/*/*/*.bbappend \
     tpm-layer:${LAYERDIR}/dynamic-layers/tpm-layer/*/*/*.bb \
     tpm-layer:${LAYERDIR}/dynamic-layers/tpm-layer/*/*/*.bbappend \
+    clang-layer:${LAYERDIR}/dynamic-layers/clang-layer/*/*/*.bb \
+    clang-layer:${LAYERDIR}/dynamic-layers/clang-layer/*/*/*.bbappend \
```

Comment:
The GCC 13 is only available on oe-core master branch, is this patch need on kirkstone?

Reply:
> The GCC 13 is only available on oe-core master branch, is this patch need on kirkstone?

@quaresmajose yes, it's obviously kirkstone branch of oe-core:

```
6e0d694ea1 uninative: Upgrade to 4.0 to include latest gcc 13.1.1
dbf2a81d42 uninative: Upgrade to 3.10 to support gcc 13
we're here => 18a9596ae7 selftest/reproducible: Allow native/cross reuse in test
```
We're just in 1 commit off the gcc13 in kirkstone :)